app/javascript/dashboard/components/widgets/WootWriter/Editor.vue

```diff
@@ -409,6 +436,52 @@ export default {
         },
       });
     },
+    isEditorMouseFocusedOnAnImage() {
+      const selectedNode = document.querySelector(
+        'img.ProseMirror-selectednode'
+      );
+      if (selectedNode) {
+        this.selectedNode = selectedNode;
+        this.isImageSelected = !!selectedNode;
+        // Get the position of the selected node
+        this.setToolbarPosition();
+      } else {
+        this.isImageSelected = false;
+      }
+    },
+    setToolbarPosition() {
+      const editorRoot = document.querySelector('.editor-root');
+      const editorRect = editorRoot.getBoundingClientRect();
+      const rect = this.selectedNode.getBoundingClientRect();
+      this.toolbarPosition = {
+        top: `${rect.top - editorRect.top - 30}px`,
+        left: `${rect.left - editorRect.left - 4}px`,
+      };
+    },
+    setURLWithQueryAndSize(size) {
+      if (this.selectedNode) {
+        // Update the URL query "cw_image_height"
+        const url = new URL(this.selectedNode.src);
+        // Set the new URL with the updated query with the height
+        url.searchParams.set('cw_image_height', size.height);
+        this.selectedNode.src = url.href;
+        this.selectedNode.style.height = size.height;
```

Comment:
Can we move this to a separate util, perhaps someplace like `editorHelper.js` that way we can unit test this